sdk/program/Cargo.toml

```diff
@@ -73,6 +72,7 @@ console_error_panic_hook = { workspace = true }
 console_log = { workspace = true }
 getrandom = { workspace = true, features = ["js", "wasm-bindgen"] }
 js-sys = { workspace = true }
+rand = { version = "0.7.3", features = ["wasm-bindgen"] }
```

Comment:
`version = workspace`? Also, does this need to be added if `tiny-bip39` comes out?